cmd/admin-handlers.go

```diff
@@ -54,52 +54,69 @@ const (
 	mgmtUploadID       mgmtQueryKey = "upload-id"
 )
 
-// ServerVersion - server version
-type ServerVersion struct {
-	Version  string `json:"version"`
-	CommitID string `json:"commitID"`
-}
+var (
+	// This struct literal represents the Admin API version that
+	// the server uses.
+	adminAPIVersionInfo = madmin.AdminAPIVersionInfo{"1"}
+)
+
+// VersionHandler - GET /minio/admin/version
+// -----------
+// Returns Administration API version
+func (adminAPI adminAPIHandlers) VersionHandler(w http.ResponseWriter,
+	r *http.Request) {
```

Comment:
This style of odd indentation is not needed in case of VersionHandler()  just change the `(adminAPI adminAPIHandlers)` to simply `(a adminAPIHandlers)`

Reply:
Done.